src/pages/home/report/ReportActionItem.tsx

```diff
@@ -486,6 +488,8 @@ function ReportActionItem({
                 onPress: () => Report.resolveActionableMentionWhisper(report.reportID, action, CONST.REPORT.ACTIONABLE_MENTION_WHISPER_RESOLUTION.NOTHING),
             },
         ];
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why did you disable this lint rule?

Reply:
This is outdated, we don't need this anymore.